src/components/formWeekday/formWeekday.tsx

```diff
@@ -0,0 +1,72 @@
+import { Form } from "../form/form";
+import { useForm } from "react-hook-form";
+import React from "react";
+
+type WeekdaysProps = {
+  title: string;
+  hintText: string;
+};
+
+const FormWeekday = ({ title, hintText }: WeekdaysProps) => {
+  const { register, handleSubmit, setError, clearErrors } = useForm();
+  const [currentSelectedDays, setCurrentSelectedDays] = React.useState<
+    string[]
+  >([]);
+
+  const days = ["Mon", "Tue", "Wed", "Thur", "Fri", "Sat", "Sun"];
+
+  const onSubmit = (data: object) => {
+    if (currentSelectedDays.length === 0) {
+      setError("selectedDays", {
+        message: "Please select a day",
+      });
+    } else {
+      clearErrors("selectedDays");
+      console.log(data);
+    }
+  };
+
+  const handleCheck = (day: string) => {
+    if (currentSelectedDays.includes(day)) {
+      setCurrentSelectedDays(currentSelectedDays.filter((d) => d != day));
+    } else {
+      setCurrentSelectedDays([...currentSelectedDays, day]);
+    }
+  };
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <div className="max-w-pageContent mx-3 relative font-text">
+        <label className="">{title}</label>
```

Comment:
I would add the proper font to all of the text in this component. Figma calls fore Rubik which would be ```font-text```

Reply:
I added it on the parent <div> wrapper so I wouldn't have to re-write for each component. Do you think I should add the font-text individually?